core/src/main/scala/filodb.core/metadata/IngestionState.scala

```diff
@@ -0,0 +1,25 @@
+package filodb.core.metadata
+
+trait IngestionState {
```

Comment:
I don't think we need a separate trait for this, probably can just collapse into a single case class